doc/2/controllers/security/delete-role/snippets/delete-role.dart

```diff
@@ -0,0 +1,12 @@
+final result = await kuzzle.security.deleteRole('myRole');
+/*
+{ found: true,
+  _index: '%kuzzle',
+  _type; 'roles',
+  _id: 'myRole',,
+  _version: 2,
+  result: 'deleted',
+  _shards: {  total: 2, successful: 1, failed: 0 } }
+  */
+
+print('Role successfully deleted');
```

Comment:
Please do the print in the template by using the template `default` or `print-result` for example.